sanic/config.py

```diff
@@ -192,7 +221,7 @@ def load_environment_vars(self, prefix=SANIC_PREFIX):
 
             _, config_key = key.split(prefix, 1)
 
-            for converter in (int, float, str_to_bool, str):
+            for converter in self.__registry__:
```

Comment:
This will change the behaviour described in the documentation of the method 👀

Reply:
How so?

Reply:
There is no other change, i think, but it might be nice to update the documentation too while its being changed?